src/entities/discussion/discussion.model.ts

```diff
@@ -0,0 +1,73 @@
+import * as DB from '@nestjs/mongoose' // { Prop, Schema, SchemaFactory }
+import * as GQL from '@nestjs/graphql' // { Field, ObjectType, ID }
+import mongoose from 'mongoose'
+import autopopulate from 'mongoose-autopopulate'
+import * as Utils from '../../utils/Model'
+import * as V from 'class-validator' // { Prop, Schema, SchemaFactory }
+
+// Pagination
+import { Connected, ConnectionType, withCursor } from '../../utils/Connection'
+
+// Entities
+import { Subject } from '../subject/subject.model'
+import { CalendarEvent } from '../calendarEvent/calendarEvent.model'
+import { User, ConnectedUsers } from '../user/user.model'
+
+// Embedded documents
+import { Comment, CommentSchema } from '../comment/comment.model'
+
+@GQL.ObjectType()
+@DB.Schema({
+  autoIndex: true,
+  timestamps: true,
+})
+export class Discussion extends Utils.BaseModel {
+  constructor(subject: Subject) {
+    super()
+    this.subject = subject
+  }
+
+  @GQL.Field(() => GQL.ID)
+  _id: mongoose.Types.ObjectId
+
+  @GQL.Field(() => String, { nullable: true })
+  @DB.Prop(() => String)
+  title: string
+
+  @GQL.Field(() => String, { nullable: true })
```

Comment:
why is it nullable? may you make it false and add a validation of at lease 2 chars?